### PR TITLE
Modal

### LINKED ISSUE #8: Modal/Dialogs
- [x] Alert dialog
image
- [x] Simple dialog
image
- [x] Confirmation dialog
image

### PR DESCRIPTION
**Pull request by mohitdhatrak**
Deploy preview: https://deploy-preview-24--spark-ui.netlify.app/components/modal/modal

# Modal

image

image

image

image

image

Closes #8

### CODE REVIEW

File: Components/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+    /* flex - properties */
+    display: flex;
+    flex-direction: column;
+    gap: 1em;
+    align-items: center;
+}
+
+.modal.simple-modal {
+    gap: 0;
```
Comment (irajbhinde) [inline Components/Modal/modal.css:43]: Looked into the deploy preview & did not see gap 0 making any significant change. Why was this required?

File: Components/Modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+}
+
+.modal.confirmation-modal {
+    gap: 0;
```
Comment (irajbhinde) [inline Components/Modal/modal.css:47]: Same as above

**Review by irajbhinde — COMMENTED** [review]
Looks good, just check if that gap is required.

File: Components/Modal/modal.css (hunk 1 above)
Comment (mohitdhatrak) [reply Components/Modal/modal.css:43]: Actually it does make a small difference in simple modal when u scroll there is a gap that doesn't look good, u can try using inspect to see the difference

**Comment by mohitdhatrak** [comment]
> Looks good, just check if that gap is required.

@irajbhinde I can show in vc later why gap 0, it was overriding the previous gap

**Review by sanjay-xdr — APPROVED** [review]
everything looks good
